packages/storage-legacy/src/LegacyAsyncStorage.ts

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {NativeModules} from 'react-native';
+import {IStorageBackend, StorageOptions} from '../../core/types';
```

Comment:
We need to treat each package as a separate module, therefore we need to import those types from `@react-native-community/async-storage`.